client/src/types.ts

```diff
@@ -167,6 +172,7 @@ export enum QuestionType {
 }
 
 export enum UtteranceName {
+  NONE = "NONE",
```

Comment:
is this `NONE` coming from the server? I actively hate the idea of a NONE string instead of the empty string. 

If that's how it already works in the server/data, then let's at least limit the scope of handling the NONE string to a single helper function, e.g.

```typescript
export function isNoneUtteranceName(n: UtteranceName): boolean {
  return n && n !== UtteranceName.NONE
}
```